openmdao/docs/_utils/docutil.py

```diff
@@ -662,7 +662,13 @@ def get_test_src(method_path):
             save_opts = np.get_printoptions()
             np.set_printoptions(precision=8)
 
+            # Move to the test directory in case there are files to read.
+            save_dir = os.getcwd()
+            os.chdir('/'.join(test_module.__file__.split('/')[:-1]))
+
             exec(code_to_run, {})
+
+            os.chdir(save_dir)
```

Comment:
maybe wrap the exec in a try-finally block to make sure that no matter what you get back to the original directory.

Reply:
Fixed